docs/windows.md

```diff
@@ -59,7 +59,43 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (containerd based runtimes)
+
+First install antrea (v0.13.0+ is required for containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will setup antrea as a Windows service.
+
+```bash
+$TAG = "v0.13.0"
+
+curl.exe -LO "https://raw.githubusercontent.com/vmware-tanzu/antrea/${TAG}/hack/windows/Helper.psm1"
+curl.exe -LO "https://github.com/vmware-tanzu/antrea/releases/download/${TAG}/antrea-agent-windows-x86_64.exe"
```

Comment:
```
curl.exe -LO "https://github.com/vmware-tanzu/antrea/releases/download/${TAG}/antrea-agent-windows-x86_64.exe"
mv antrea-agent-windows-x86_64.exe c:/k/antrea/bin/antrea-agent.exe
```
could be removed because `Install-AntreaAgent` will download `antrea-agent-windows-x86_64.exe`.

Reply:
Thanks for reminding. Removed.